DependencyInjection/CmfSeoExtension.php

```diff
@@ -252,29 +258,48 @@ private function loadSitemapHandling($config, XmlFileLoader $loader, ContainerBu
         $loader->load('sitemap.xml');
 
         $configurations = $config['configurations'];
-        // if there are no explicit configurations, enable the default sitemap
-        if (!count($configurations)) {
-            $configurations['sitemap'] = array();
```

Comment:
moved to configuration class so that we get defaults set up by the configuration logic